### ISSUE TITLE
RANDOM UNPREDICTIBLE CRASH

### ISSUE THREAD

**Issue by Camawama** [Budschie/BudschieMorphMod#114]
My modpack has quite a few mods but your mod seemed to be working just fine until I opened the Morph Menu GUI for a second time. 

CRASH FILE
https://pastebin.com/B3hTrSsH

**Comment by Budschie** [comment]
I'm working on it :+1:

**Comment by Budschie** [comment]
I was able to identify the problem. This crash occurs when you press the key to favourite a morph whilst having the red X selected in the morph menu. 

In version `1.18.2-1.0.2`, this would simply write a message to stdout (meaning that a message would appear in the log), but this used `System.out.println`, which I had tried to get rid off in this new release in favour of using Log4J. However, in this particular case, instead of logging this, I simply instructed the mod to throw an error (because I was not aware that this edge case could actually happen).

Anyways, I have fixed this in commit dab5919a59594c50e02a3d01590c6247b65e4802. I will release a quickfix soon.
Thanks for your help :)
